src/components/Book.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+function BookList() {
+  const [books, setBooks] = useState([]);
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const newBook = { title, author };
+    setBooks([...books, newBook]);
+    setTitle('');
+    setAuthor('');
+  };
```

Comment:
* Hello 👋 @PedroH-Neves as the previous CRs were trying to explain, the requirements expect you to have two major components that for example 
* Books and Categories 
* Then inside the Books component you should have two child components 
* NewBookForm: This should be a component on its own that gets imported into the Books component the newBookForm Renders a form with inputs for the title and the author of the book
Has a button to submit( just a placeholder for now )
* Book: This component gets imported into the Books component Takes the book data as a prop and renders the title and the author, Has a button for removing books( just a placeholder for now )
* Kindly make this adjustment sorry for the long review process